torch/csrc/Exceptions.h

```diff
@@ -92,6 +92,11 @@ static inline void PyErr_SetString(PyObject* type, const std::string& message) {
     PyErr_SetString(THPException_LinAlgError, torch::processErrorMsg(msg));  \
     retstmnt;                                                                \
   }                                                                          \
+  catch (const c10::CUDAOutOfMemoryError& e) {                               \
+    auto msg = e.what();                                                     \
+    PyErr_SetString(THPException_LinAlgError, torch::processErrorMsg(msg));  \
```

Comment:
`THPException_LinAlgError` is probably not the right value.

Reply:
oops this is copy paste vestige, it should probably be `PyExc_RuntimeError`.